src/store/middlewares/voting.js

```diff
@@ -0,0 +1,23 @@
+import actionTypes from '../../constants/actions';
+import votingConst from '../../constants/voting';
+import { errorToastDisplayed } from '../../actions/toaster';
+
+const votingMiddleware = store => next => (action) => {
+  next(action);
+  if (action.type === actionTypes.voteToggled) {
+    const { votes } = store.getState().voting;
+    const voteCount = Object.keys(votes).filter(
+      key => votes[key].confirmed !== votes[key].unconfirmed).length;
+    const currentVote = votes[action.data.username] || { unconfirmed: true, confirmed: false };
+    console.log(voteCount, votingConst.maxCountOfVotesInOneTurn, currentVote);
```

Comment:
Remove `console.log`. It's interesting that you have a log here and `eslint` doesn't fail